README.md

```diff
@@ -86,11 +86,13 @@ The `liberty-maven-plugin` provides the following goals.
 | Goal | Description |
 | --------- | ------------ |
 | [install-server](docs/install-server.md#install-server) | Installs the Liberty runtime. This goal is implicitly invoked by all the other plug-in goals and usually does not need to be executed explicitly. |
-| [create-server](docs/create-server.md#create-server) | Create a Liberty server. |
-| [start-server](docs/start-server.md#start-server) | Start a Liberty server in background. The server instance will be automatically created if it does not exist. |
-| [run-server](docs/run-server.md#run-server) | Start a Liberty server in foreground. The server instance will be automatically created if it does not exist. |
-| [stop-server](docs/stop-server.md#stop-server) | Stop a Liberty server. The server instance must exist and must be running. |
-| [package-server](docs/package-server.md#package-server) | Package a Liberty server. |
+| [create-server](docs/create-server.md#create-server) | Create a Liberty Profile server. |
```

Comment:
My changes didn't get merged correctly.  I had removed "Profile" from the terminology and those changes were lost in multiple places.